lib/Migration/DataInitializationStep.php

```diff
@@ -26,5 +32,13 @@ public function run(IOutput $output): void {
 		} else {
 			$output->warning('Failed to initialize data for App Ecosystem V2');
```

Comment:
maybe here make it an `error` level, as it makes AppEcosystem unusable.

Reply:
There is no error message type in `IOutput` [interface](https://github.com/nextcloud/server/blob/master/lib/public/Migration/IOutput.php), it's used to write messages in CLI [during installation](https://github.com/nextcloud/server/blob/aba3f99e5efa02c33fec00c55b207769762cc976/lib/private/Installer.php#L165).